### PR TITLE
m_mykytovych IO 

### PR DESCRIPTION
**Pull request by marjana07**

### CODE REVIEW

File: src/main/java/iotask/Main.java [hunk 1]
```diff
@@ -0,0 +1,15 @@
+package iotask;
+
+import java.io.IOException;
+
+public class Main {
+    public static void main(String[] args) throws IOException {
```
Comment (AqaCoreSchool) [inline src/main/java/iotask/Main.java]: its better to handle exception directly in method which throws it

File: src/main/java/iotask/Writer.java [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+        }
+    }
+
+    public static XSSFColor setRandomColor() {
+        return new XSSFColor(new java.awt.Color((int) (random() * 255), (int) (random() * 255), (int) (random() * 255)));
+    }
+
+    public static void writeIntoExcel(List<String> records, String path) throws IOException {
+
+        try (Workbook book = new XSSFWorkbook()) {
```
Comment (AqaCoreSchool) [inline src/main/java/iotask/Writer.java:37]: why not catch exception here?

File: src/main/java/iotask/Main.java (hunk 1 above)
Comment (marjana07) [reply src/main/java/iotask/Main.java]: fixed

File: src/main/java/iotask/Writer.java (hunk 2 above)
Comment (marjana07) [reply src/main/java/iotask/Writer.java:37]: fixed

**Review by AqaCoreSchool — APPROVED** [review]

**Pull request merged**